open-company/open-company-web
# Collapsing comments

Card: https://trello.com/c/20ylZ32D

Related PR:
- [Interaction](https://github.com/open-company/open-company-interaction/pull/42)
- [Storage](https://github.com/open-company/open-company-storage/pull/244)

To test:
- make sure posts are collapsed when there is something in the body (i check if it has at least 50 chars of html code), it's read and it has at least 1 comment
- make sure all read comments are collapsed when opening a post
- make sure new comments are never collapsed
- if a thread is new the NEW tag is visible only on the root comment
- if more than 1 replies are new make sure all of them have the NEW tag
- check green bg has some space btw replies
- open a post with some new comments
- check a thread is marked as read when you reply
- check a single comment is marked as read when you react to it
- check new comments are still NEW when a new comment comes in
- check new comments are still NEW when you add a reply
- check new comments are not new after you go back and reopen the post
- regression test: make sure your replies are not collapsed/hidden when a comment comes in live
